kola/tests/docker/docker.go

```diff
@@ -55,12 +55,11 @@ type simplifiedDockerInfo struct {
 
 func init() {
 	register.Register(&register.Test{
-		Run:           dockerSELinux,
-		ClusterSize:   1,
-		Name:          "docker.selinux",
-		Distros:       []string{"cl"},
-		Channels:      []string{"alpha", "beta"},
-		Architectures: []string{"amd64"},
+		Run:         dockerSELinux,
+		ClusterSize: 1,
+		Name:        "docker.selinux",
+		Distros:     []string{"cl"},
+		Channels:    []string{"alpha", "beta"},
```

Comment:
While at it we should switch this to using `MinVersion` or we will forget this when the next channel transition happens

Reply:
`MinVersion:       semver.Version{Major: 2942},` should mean everything newer, including the current beta?

Reply:
why not `MinVersion: semver.Version{Major: 2984},` ? SELinux changes will be available only in the next `alpha` :thinking:

Reply:
Ah yes, I just thought about the existing `beta` value

Reply:
This is an edge case actually:
  * SELinux `amd64` is available on `alpha`/`beta`
  * SELinux `arm64` is only on `main` atm

so:
* `MinVersion: semver.Version{Major: 2942},`: will make the test fail on `arm64` for both beta / alpha
* `MinVersion: semver.Version{Major: 2984},`: will make the test disappears for `amd64` for both beta / alpha

Reply:
Yes, `MinVersion: semver.Version{Major: 2984},` is the right one :D

Reply:
Better to skip the tests than have them fail when doing a bugfix release

Reply:
ah, wait, didn't get it fully when reading the first time… we have no way to specify the version per architecture, complicated. I think it's ok to let it fail for arm64 on Alpha bugfix releases then and use `2942`.

Reply:
ok done with `2942` :)